Wikipedia/UI-V5/WMFAppViewController.m

```diff
@@ -362,4 +360,14 @@ - (void)searchLanguageDidChangeWithNotification:(NSNotification*)note {
     [self configureSearchViewController];
 }
 
+#pragma mark - UINavigationControllerDelegate
+
+- (void)navigationController:(UINavigationController*)navigationController
+      willShowViewController:(UIViewController*)viewController
+                    animated:(BOOL)animated {
+    BOOL isToolbarEmpty = [viewController toolbarItems].count == 0;
+    [self.rootTabBarController wmf_setTabBarVisible:isToolbarEmpty animated:NO completion:nil];
```

Comment:
I actually forgot, `hidesBottomBarWhenPushed` is being set on the article container... let me fiddle with this a bit and see if there's a clear reason why it works the way it does when setup this way and if there's a less-intrusive way to do it.